sourcegraph/sourcegraph
# notebooks & embed: open links in a new tab

Fixes #30731

Open search result file matches in a new tab in notebook query block. Also, open all links in a new tab when embedded inside an iframe.

## Test plan



* Open notebook, run a search, click on a file match result -> should open in a new tab
* Check in Chrome, Firefox, Safari
* Embed a notebook inside an iframe, click on any link inside an embedded iframe, it should open in a new tab
* Or, navigate to local-dev-env.test/embed/notebooks/abc, and click on any link, it should also open in a new tab
* Check in Chrome, Firefox, Safari
